### DISCUSSION TITLE
zh/docs/master/gorse-dashboard

### DISCUSSION THREAD

**Discussion by giscus** [gorse-io/docs#115] [Giscus]
# zh/docs/master/gorse-dashboard

Gorse 控制台 Gorse 的主节点提供了 Gorse 控制台，并可以通过主节点的 HTTP 端口（默认为8088 ）访问。 HTTP 端口可在配置文件中指定。控制台可以让您观察推荐系统的当前状态，预览用户推荐结果，导入和导出数据。 登录页 默认配置允许不登录就可以访问控制台。在配置文件中可以设置登录控制台的用户名和密码。 [master] # Username for the master node dashboard. dashboard_user_name = 

https://gorse.io/zh/docs/master/gorse-dashboard.html

**Comment by Strhao123** [comment]
666
